src/hooks/useExtensions.ts

```diff
@@ -49,6 +50,10 @@ export function useExtensions(api: ApiPromise, store: any) {
 
   (async () => {
     try {
+      if (isMobileDevice) {
+        // MEMO: resolve the issue that do not sync accounts in nova wallet
+        await wait(400);
```

Comment:
is this really the only way to wait for the mobile client to initialize? Are there any other way to check if the mobile client is ready?

Reply:
I know it's hacky. but I couldn't find out the essential reason since the issue is only caused at nova wallet. Also, the wallet side also needs refactoring later. but for now, need to focus on what works